webrtc.html

```diff
@@ -5396,10 +5396,87 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
+    application running in the same browser and something that is not a
+    browser, something that is an extension to the usual barriers in
+    the Web model against sending data between entities with different
+    origins.</p>
+
+    <p>The WebRTC specification provides no user prompts or chrome
+    indicators for communication; it assumes that once the Web page
+    has been allowed to access media, it is free to share that media
+    with other entities as it chooses.</p>
+
+    <p>A mechanism, <code><a>peerIdentity</a></code>, is provided that gives
+    Javascript the option of requesting media that the same javascript
+    cannot access, but can only be sent to certain other entities.</p>
+
+    <p>Even without WebRTC, the Web server providing a Web application
+    will know the public IP
+    address to which the application is delivered. Setting up
+    communications exposes additional information about the browser’s
+    network context to the web application, including the set of
+    (possibly private) IP addresses available to the browser for WebRTC
+    use. Some of this information has to be passed to the
+    corresponding party to enable the establishment of a communication
+    session.</p>
+
+    <p>Revealing IP addresses can leak location and means of
+    connection; this can be sensitive.</p>
+
+    <p>Mitigating the exposure of IP addresses to the application requires
+    limiting the IP addresses that can be used, which will impact the
+    ability to communicate on the most direct path between endpoints.
+    Browsers are encouraged to provide appropriate controls for deciding
+    which IP addresses are made available to applications, based on the
+    security posture desired by the user.
+
+    <p>A connection will always reveal the IP addresses proposed for
+    communication to the corresponding party. The application can limit
+    this exposure by choosing not to use certain addresses using the
+    <a>RTCIceTransportPolicy</a>, and by
+    using relays (for instance TURN servers) rather than direct
+    connections between participants. One will normally assume that
+    the IP address of TURN servers is not sensitive information.</p>
+
+    <p>Since the browser is an active platform executing in a trusted
+    network environment (inside the firewall), it is important to
+    limit the damage that the browser can do to other elements on the
+    local network, and it is important to protect data from
+    interception, manipulation and modification by untrusted
+    participants.</p>
+
+    <p>Mitigations include:</p>
+    <ul>
+      <li>Always requesting permission to communicate using ICE. This
+      ensures that the browser can only send to partners who you have
+      shared credentials with.</li>
+      <li>Requesting ongoing permission using ICE continued
+      consent. This enables a receiver to withdraw consent to
+      receive.</li>
```

Comment:
So probably need to explain the consent is the media consent stuff and not a prompt to user.